### PR TITLE
Objects Automated Test - LPS-141663 | master

### PR DESCRIPTION
**Pull request by andreefarias**
https://issues.liferay.com/browse/LPS-141663

### CODE REVIEW

**Comment by andreefarias** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase [hunk 1]
```diff
@@ -3992,12 +3992,109 @@ definition {
...
+			relationshipLabel = "Relationship A",
+			relationshipName = "relationshipA",
+			relationshipType = "manyToMany");
+
+		ObjectAdmin.addObjectRelationshipViaAPI(
+			objectName_1 = "CustomObjectB",
+			objectName_2 = "CustomObjectA",
+			relationshipLabel = "Relationship B",
+			relationshipName = "relationshipB",
+			relationshipType = "manyToMany");
```
Comment (evanilson-santana) [inline modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase:4029]: Hi @andreefarias 

You don't need to create this relation for this step.

**Review by evanilson-santana — CHANGES_REQUESTED** [review]
Hi @andreefarias 

Please, check my comments and make the necessary changes.

Thanks!!!

**Comment by andreefarias** [comment]
ci:test:sf

**Review by evanilson-santana — APPROVED** [review]

**Comment by evanilson-santana** [comment]
ci:forward

**Pull request closed without merging**